src/main/java/teammates/common/datatransfer/attributes/CourseAttributes.java

```diff
@@ -87,6 +87,9 @@ public Instant getCreatedAt() {
     }
 
     public Instant getDeletedAt() {
+        if (this.deletedAt == null) {
```

Comment:
This conversion should be inside response DTO? Put it here will cause a lot of regression.